### PR TITLE
Created result view and implemented Lottie

### PR DESCRIPTION
**Pull request by gciadiego**

### CODE REVIEW

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+    var viewTitle = ""
+    var viewColor = Snow
+    var viewAnimation = correctAnswerComposition
+    var animationSize = 400.dp
+    var viewMessage = ""
+    var bottomText = ""
+
+    if(!gameOver){
+        viewTitle = "Ronda #$roundNumber"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        bottomText = "Esperando al administrador..."
+        if(roundWinner == userId){
+            viewColor = ShamrockGreen
+            viewAnimation = correctAnswerComposition
+            viewMessage = "¡Respuesta correcta!"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            animationSize = 400.dp
+        }else{
+            viewColor = RedSalsa
+            viewAnimation = wrongAnswerComposition
+            viewMessage = "¡Punto para "+score.filter { it.second == roundWinner }.map { it.first }+"!"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+            animationSize = 400.dp
+        }
+    }else{
+        viewTitle = "Resultados"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+        bottomText = "Ir a inicio"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+        if(score.sortedBy { it.third }.last().second == userId){
+            viewColor = ShamrockGreen
+            viewAnimation = winnerComposition
+            viewMessage = "¡Ganaste!"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+            animationSize = 400.dp
+        }else{
+            viewColor = RedSalsa
+            viewAnimation = null
+            viewMessage = "¡Perdiste!"
```
Comment (dmroldanch) [inline presentation/src/main/java/com/kk/presentation/ResultView.kt]: You have to generate a variable for the string in a String file

File: presentation/src/main/java/com/kk/presentation/ResultView.kt (hunk 1 above)
Comment (gciadiego) [reply presentation/src/main/java/com/kk/presentation/ResultView.kt]: Fixed

**Pull request closed without merging**